File: examples/send.php
<?php
function alarm_message($message) {
    $alarm = 'http://t69.me';
    $key = '71b6ffb3-7cff-403e-a2f7-15e3d919f1e1';

    $data = json_encode(['key' => $key, 'message' => $message]);

    $options = [
        'http' => [
            'header'  => "Content-Type: application/json\r\n",
            'method'  => 'POST',
            'content' => $data,
        ]
    ];

    $context  = stream_context_create($options);
    $response = file_get_contents($alarm, false, $context);

    if ($response === FALSE) {
        error_log("Alarm request failed");
    } else {
        echo "Alarm sent successfully";
    }
}

alarm_message("😀 Hello World");
?>

File: examples/send.py
import requests
from loguru import logger


def alarm_message(message: str) -> None:
    alarm = 'http://t69.me'
    key = "71b6ffb3-7cff-403e-a2f7-15e3d919f1e1"

    try:

        response = requests.post(alarm, json={'key': key, 'message': message})

        # Optional: Log the response if needed
        if response.status_code != 200:
            logger.error(f"Alarm request failed with status {response.status_code}: {response.text}")
        else:
            logger.info("Alarm sent successfully")
    except Exception as ex:
        logger.error(f'Alarm error for {key}: {str(type(ex))}')


alarm_message(message=" 😀 Hello World ")
